### ISSUE TITLE
Feature request: see posts I upvoted

### ISSUE THREAD

**Issue by InCogNiTo124** [SimonHalvdansson/Harmonic-HN#172]
HN on web allows me to see eveything I upvoted using this: https://news.ycombinator.com/upvoted?id=<user_id>

**Comment by flofriday** [comment]
As far as I can remember the API doesn't provide that functionality. 

So the only way to do that would be to parse HTML which has it's own troubles.

**Comment by SimonHalvdansson** [comment]
Yeah I think so too, however the HTML parsing is not necessarily crazy hard. For the short lived Nitter integration I wrote a class to do this which worked well so I think this issue should stay open.

**Comment by SimonHalvdansson** [comment]
https://github.com/SimonHalvdansson/Harmonic-HN/commit/92f5715c215995277ed01f8de9feff5b778881cb
